.github/workflows/sycl-linux-precommit.yml

```diff
@@ -92,6 +92,11 @@ jobs:
             image_options: -u 1001 --device=/dev/dri -v /dev/dri/by-path:/dev/dri/by-path --privileged --cap-add SYS_ADMIN
             target_devices: level_zero:gpu;opencl:gpu
             extra_lit_opts: -j 50
+          - name: E2E tests on Intel Ponte Vecchio GPU for L0 V2
```

Comment:
PVC runners are kind of a bottleneck in Linux precommit now. Can you consider using some other runner (Gen12/Arc/BMG (once it's stabilized))?

Reply:
Sure, I can enable that only for Arc for now

Reply:
> PVC runners are kind of a bottleneck

We already migrated two PVC runners into intel/llvm (one for perf, the other for UR L0 CI jobs), and are planning to add another. There's going to be a bit of work to make them usable for other intel/llvm jobs, but I'll ask the team to look into that so we can more efficiently utilize our resources.

Reply:
I switched to using Arc only for now